google-cloud-tools-plugin/build.gradle

```diff
@@ -27,6 +27,7 @@ intellij {
     publish {
         pluginId '8079'
     }
+    systemProperties = ['compiler.process.debug.port':56666]
```

Comment:
The build flow happens in another process. When run in debug mode, it listens to a random port. This makes it always listen to the port 56666, so I don't have to constantly change the port in my run config every time I want to debug the build process.